### DISCUSSION TITLE
How do i set timer for lockout? How to make my OS still running if i want to close my laptop lid?

### DISCUSSION THREAD

**Discussion by blutub** [prasanthrangan/hyprdots#879] [Q&A]
I've uncomment the line on the hyprland.conf file to set the timeout for lock. But it didn't seem to work. 
I also want to ask how to make so if i close my laptop lid, i can still use the external monitor as my primary one. ATM if i close the lid, it will glitched so hard that i can't do anyting

**Comment by prasanthrangan** [comment]
Hi, please try commenting out this line in `~/.config/hypr/keybindings.conf` --

https://github.com/prasanthrangan/hyprdots/blob/5bb3e66389eb7fe8141c4f9a88840bb6c66c0814/Configs/.config/hypr/keybindings.conf#L141
